Help Command does not work when GCTL_SESSION_ID is not set

**What happened**:
I executed `gardenctl` on windows power shell and got the following error:

```ps
PS C:\Users\xxx> gardenctl version
panic: environment variable GCTL_SESSION_ID is required. Use "gardenctl help" for more information about the requirements of gardenctl

goroutine 1 [running]:
k8s.io/apimachinery/pkg/util/runtime.Must(...)
        /home/runner/go/pkg/mod/k8s.io/apimachinery@v0.25.0/pkg/util/runtime/runtime.go:172
github.com/gardener/gardenctl-v2/pkg/cmd/ssh.NewCmdSSH({0x26686a8?, 0xc00054c720}, 0xc00043e580)
        /home/runner/work/gardenctl-v2/gardenctl-v2/pkg/cmd/ssh/ssh.go:47 +0x351
github.com/gardener/gardenctl-v2/pkg/cmd.NewGardenctlCommand(0xc00054c720, {{0x2648280, 0xc000146000}, {0x26482c0, 0xc000146008}, {0x26482c0, 0xc000146010}})
        /home/runner/work/gardenctl-v2/gardenctl-v2/pkg/cmd/cmd.go:115 +0x45d
github.com/gardener/gardenctl-v2/pkg/cmd.NewDefaultGardenctlCommand()
        /home/runner/work/gardenctl-v2/gardenctl-v2/pkg/cmd/cmd.go:57 +0x130
github.com/gardener/gardenctl-v2/pkg/cmd.Execute()
        /home/runner/work/gardenctl-v2/gardenctl-v2/pkg/cmd/cmd.go:45 +0x19
main.main()
        /home/runner/work/gardenctl-v2/gardenctl-v2/main.go:23 +0x6d
```

However, the command `gardenctl help` produces the same error as above.

**What you expected to happen**:
Commands that do not require `GCTL_SESSION_ID` to be set (help, version, ...) should not produce this error.

**How to reproduce it (as minimally and precisely as possible)**:
Install gardenctl on windows power shell (e.g. via choco). Run arbitrary gardenctl command.

**Anything else we need to know**:
Workaround: As the help command currently does not work, you can find the information how to set `GCTL_SESSION_ID` here: https://github.com/gardener/gardenctl-v2#shell-session

**Environment**:
PowerShell @windows10

--- a/pkg/cmd/cmd_test.go
+++ b/pkg/cmd/cmd_test.go
@@ -7,6 +7,7 @@ SPDX-License-Identifier: Apache-2.0
 package cmd_test
 
 import (
+	"os"
 	"path/filepath"
 	"strings"
 
@@ -78,6 +79,37 @@ var _ = Describe("Gardenctl command", func() {
 				Expect(current.SeedName()).To(BeEmpty())
 				Expect(current.ShootName()).To(Equal(shootName))
 			})
+		})
+	})
+
+	Context("when running the help command", func() {
+		var (
+			sessionID     string
+			termSessionID string
+		)
+
+		BeforeEach(func() {
+			sessionID = os.Getenv("GCTL_SESSION_ID")
+			termSessionID = os.Getenv("GCTL_SESSION_ID")
+
+			Expect(os.Unsetenv("GCTL_SESSION_ID")).To(Succeed())
+			Expect(os.Unsetenv("TERM_SESSION_ID")).To(Succeed())
+		})
+
+		AfterEach(func() {
+			// restore env variables
+			Expect(os.Setenv("GCTL_SESSION_ID", sessionID)).To(Succeed())
+			Expect(os.Setenv("TERM_SESSION_ID", termSessionID)).To(Succeed())
+		})
+
+		It("should succeed without session IDs", func() {
+			args := []string{
+				"help",
+			}
+
+			cmd := cmd.NewGardenctlCommand(util.NewFactoryImpl(), streams)
+			cmd.SetArgs(args)
+			Expect(cmd.Execute()).To(Succeed())
 		})
 	})
 })

--- a/pkg/cmd/cmd_test.go
+++ b/pkg/cmd/cmd_test.go
@@ -90,7 +90,7 @@ var _ = Describe("Gardenctl command", func() {
 
 		BeforeEach(func() {
 			sessionID = os.Getenv("GCTL_SESSION_ID")
-			termSessionID = os.Getenv("GCTL_SESSION_ID")
+			termSessionID = os.Getenv("TERM_SESSION_ID")
 
 			Expect(os.Unsetenv("GCTL_SESSION_ID")).To(Succeed())
 			Expect(os.Unsetenv("TERM_SESSION_ID")).To(Succeed())

--- a/pkg/cmd/env/env.go
+++ b/pkg/cmd/env/env.go
@@ -11,7 +11,6 @@ import (
 	"runtime"
 
 	"github.com/spf13/cobra"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	"github.com/gardener/gardenctl-v2/internal/util"
 	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
@@ -57,9 +56,7 @@ here https://github.com/gardener/gardenctl-v2/tree/master/pkg/cmd/env/templates.
 	persistentFlags := cmd.PersistentFlags()
 	o.AddFlags(persistentFlags)
 
-	manager, err := f.Manager()
-	utilruntime.Must(err)
-	manager.TargetFlags().AddFlags(persistentFlags)
+	f.TargetFlags().AddFlags(persistentFlags)
 	flags.RegisterCompletionFuncsForTargetFlags(cmd, f, ioStreams, persistentFlags)
 
 	for _, s := range validShells {

--- a/pkg/cmd/env/env_test.go
+++ b/pkg/cmd/env/env_test.go
@@ -35,6 +35,7 @@ var _ = Describe("Env Commands", func() {
 		factory.EXPECT().Manager().Return(manager, nil).AnyTimes()
 
 		targetFlags := target.NewTargetFlags("", "", "", "", false)
+		factory.EXPECT().TargetFlags().Return(targetFlags).AnyTimes()
 		manager.EXPECT().TargetFlags().Return(targetFlags).AnyTimes()
 
 		streams = util.IOStreams{}

--- a/pkg/cmd/kubeconfig/kubeconfig.go
+++ b/pkg/cmd/kubeconfig/kubeconfig.go
@@ -48,9 +48,7 @@ gardenctl kubeconfig --garden my-garden --project my-project`,
 
 	o.AddFlags(cmd.Flags())
 
-	manager, err := f.Manager()
-	utilruntime.Must(err)
-	manager.TargetFlags().AddFlags(cmd.Flags())
+	f.TargetFlags().AddFlags(cmd.Flags())
 	flags.RegisterCompletionFuncsForTargetFlags(cmd, f, ioStreams, cmd.Flags())
 
 	utilruntime.Must(cmd.RegisterFlagCompletionFunc("output", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {

--- a/pkg/cmd/kubeconfig/kubeconfig_test.go
+++ b/pkg/cmd/kubeconfig/kubeconfig_test.go
@@ -68,15 +68,13 @@ var _ = Describe("Kubeconfig Command - Options", func() {
 			manager.EXPECT().ClientConfig(ctx, t).Return(config, nil)
 
 			targetFlags := target.NewTargetFlags("", "", "", "", false)
-			manager.EXPECT().TargetFlags().Return(targetFlags).AnyTimes()
+			factory.EXPECT().TargetFlags().Return(targetFlags).AnyTimes()
 
 			streams, _, out, _ = util.NewTestIOStreams()
 			cmd = cmdkubeconfig.NewCmdKubeconfig(factory, streams)
 		})
 
 		It("should execute the kubeconfig subcommand", func() {
-			factory.EXPECT().Manager().Return(manager, nil).Times(2)
-
 			cmd = cmdkubeconfig.NewCmdKubeconfig(factory, streams)
 			cmd.SetArgs([]string{"--output", "yaml"})
 			Expect(cmd.Execute()).To(Succeed())

--- a/pkg/cmd/ssh/ssh.go
+++ b/pkg/cmd/ssh/ssh.go
@@ -10,7 +10,6 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	"github.com/gardener/gardenctl-v2/internal/util"
 	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
@@ -43,9 +42,7 @@ func NewCmdSSH(f util.Factory, o *SSHOptions) *cobra.Command {
 	o.AccessConfig.AddFlags(cmd.Flags())
 	RegisterCompletionFuncsForAccessConfigFlags(cmd, f, o.IOStreams, cmd.Flags())
 
-	manager, err := f.Manager()
-	utilruntime.Must(err)
-	manager.TargetFlags().AddFlags(cmd.Flags())
+	f.TargetFlags().AddFlags(cmd.Flags())
 	flags.RegisterCompletionFuncsForTargetFlags(cmd, f, o.IOStreams, cmd.Flags())
 
 	return cmd

--- a/pkg/cmd/ssh/ssh_test.go
+++ b/pkg/cmd/ssh/ssh_test.go
@@ -282,7 +282,7 @@ var _ = Describe("SSH Command", func() {
 
 		It("should reject bad options", func() {
 			o := ssh.NewSSHOptions(streams)
-			cmd := ssh.NewCmdSSH(&util.FactoryImpl{}, o)
+			cmd := ssh.NewCmdSSH(util.NewFactoryImpl(), o)
 
 			Expect(cmd.RunE(cmd, nil)).NotTo(Succeed())
 		})

--- a/pkg/cmd/sshpatch/sshpatch.go
+++ b/pkg/cmd/sshpatch/sshpatch.go
@@ -10,7 +10,6 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	"github.com/gardener/gardenctl-v2/internal/util"
 	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
@@ -50,9 +49,7 @@ gardenctl ssh-patch cli-xxxxxxxx`,
 
 	ssh.RegisterCompletionFuncsForAccessConfigFlags(cmd, f, o.IOStreams, cmd.Flags())
 
-	manager, err := f.Manager()
-	utilruntime.Must(err)
-	manager.TargetFlags().AddFlags(cmd.Flags())
+	f.TargetFlags().AddFlags(cmd.Flags())
 	flags.RegisterCompletionFuncsForTargetFlags(cmd, f, o.IOStreams, cmd.Flags())
 
 	return cmd

--- a/pkg/cmd/sshpatch/sshpatch_test.go
+++ b/pkg/cmd/sshpatch/sshpatch_test.go
@@ -188,6 +188,8 @@ var _ = Describe("SSH Patch Command", func() {
 		ctrl = gomock.NewController(GinkgoT())
 		gardenClient = gcmocks.NewMockClient(ctrl)
 
+		targetFlags := target.NewTargetFlags("", "", "", "", false)
+
 		manager = targetmocks.NewMockManager(ctrl)
 		manager.EXPECT().ClientConfig(gomock.Any(), gomock.Any()).DoAndReturn(func(_ context.Context, _ target.Target) (clientcmd.ClientConfig, error) {
 			// DoAndReturn allows us to modify the apiConfig within the testcase
@@ -195,7 +197,6 @@ var _ = Describe("SSH Patch Command", func() {
 			return clientcmdConfig, nil
 		}).AnyTimes()
 		manager.EXPECT().CurrentTarget().Return(currentTarget, nil).AnyTimes()
-		manager.EXPECT().TargetFlags().Return(target.NewTargetFlags("", "", "", "", false)).AnyTimes()
 		manager.EXPECT().GardenClient(gomock.Eq(gardenName)).Return(gardenClient, nil).AnyTimes()
 
 		ctx, cancel = context.WithTimeout(context.Background(), 30*time.Second)
@@ -203,6 +204,7 @@ var _ = Describe("SSH Patch Command", func() {
 
 		factory = utilmocks.NewMockFactory(ctrl)
 		factory.EXPECT().Manager().Return(manager, nil).AnyTimes()
+		factory.EXPECT().TargetFlags().Return(targetFlags).AnyTimes()
 		factory.EXPECT().Context().Return(ctx).AnyTimes()
 		factory.EXPECT().Clock().Return(clock).AnyTimes()
 		fakeIPs := []string{"192.0.2.42", "2001:db8::8a2e:370:7334"}

